scripts/dist_build.sh

```diff
@@ -28,24 +33,26 @@ make get_vendor_deps
 # ldflags: -s Omit the symbol table and debug information.
 #	         -w Omit the DWARF symbol table.
 echo "==> Building..."
-"$(which gox)" \
```

Comment:
Which is better? Using gox or?

Reply:
replacing gox is 5 lines. don't think we need it